data/tlg4034/tlg006/__cts__.xml

```diff
@@ -2,6 +2,6 @@
   <ti:title xml:lang="eng">In Librum Quintum Ethicorum Nicomacheorum Commentarium</ti:title>
   <ti:edition urn="urn:cts:greekLit:tlg4034.tlg006.opp-grc1" workUrn="urn:cts:greekLit:tlg4034.tlg006">
     <ti:label xml:lang="eng">In Librum Quintum Ethicorum Nicomacheorum Commentarium</ti:label>
-    <ti:description xml:lang="eng">Michael, In Librum Quintum Ethicorum Nicomacheorum Commentarium</ti:description>
+    <ti:description xml:lang="eng">Michael, In Librum Quintum Ethicorum Nicomacheorum Commentarium, Berolini: Typis et Impensis G. Reimeri, 1903</ti:description>
```

Comment:
In the text of the `<ti:description>` element, I would make the author's name "Michael Ephesius" in order to be more clear.
Instead of using the Latin text for the city and publisher, I would make the "Berlin, Reimer".
The `xml:lang` atrtribute here should be "mul".
Also add the name of the editor here.